build-system/compile/pre-closure-babel.js

```diff
@@ -78,10 +67,8 @@ function preClosureBabel() {
       return next(null, file);
     }
 
-    const hash = sha256(file.contents);
-    const cached = cache[path];
-    if (cached && cached.hash === hash) {
```

Comment:
The old code was a second (third?) attempt at implementing a fast cache for babel transforms, written by @jridgewell and reviewed by me. IIRC, the reason we were first checking the hash was so that cached transforms would be returned only if it were guaranteed that the underlying toolchain did not change. From reading the new code in this PR, I think we're okay, because this is an in-memory cache, and it looks more elegant. However, I'm calling this out for visibility in case there's an unhandled corner case that Justin can identify.

Reply:
Should be fine. The only issues we'd face is if parts of the app were complied with a different set of babel plugins, but depended on the same file.